src/notification.cpp

```diff
@@ -420,7 +563,14 @@ void Notification::setReplacesId(uint id)
 /*!
     \qmlproperty QString Notification::appIcon
```

Comment:
QML docs should use QML property types for property docs i.e. 'string' rather than 'QString', 'int' rather than 'int32' or 'uint'